helm/dashboards/dashboards/shared/private/external-dns.json

```diff
@@ -0,0 +1,1076 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": {
+          "type": "datasource",
+          "uid": "grafana"
+        },
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
+        "name": "Annotations & Alerts",
+        "type": "dashboard"
+      }
+    ]
+  },
+  "description": "Dashboard to display external-dns metrics",
+  "editable": true,
+  "fiscalYearStartMonth": 0,
+  "gnetId": 15038,
+  "graphTooltip": 0,
+  "id": 103,
+  "links": [],
+  "liveNow": false,
+  "panels": [
+    {
+      "collapsed": false,
+      "datasource": {
+        "type": "prometheus",
+        "uid": "PBFA97CFB590B2093"
```

Comment:
I'm not sure, but didn't we use `"datasource": "default"` here? Maybe some @giantswarm/team-atlas knows.

Reply:
that is the default datasource